third_party/move/move-vm/test-utils/src/gas_schedule.rs

```diff
@@ -39,6 +35,17 @@ use std::{
     u64,
 };
 
+// gas meter for test-utils
+pub trait TestGasMeter: GasMeter {
+    fn instantiate(&self) -> Self;
+    fn remaining_gas(&self) -> Gas;
```

Comment:
`trait GasMeter` already has a `fn balance_internal(&self) -> InternalGas` method

The reason why I do not offer a `fn balance(&self) -> Gas` method is because there are different rounding methods -- only the client knows which should be used.